web/app/models.py

```diff
@@ -226,7 +227,7 @@ def __str__(self):
 class TournamentPlayer(db.Model):
```

Comment:
I was imagining a real hosted AGA tournament. Unfortunately, it's not possible to guarantee that Players exist at the time of the tournament (players can register with the AGA at the tournament). So, thus the player copy. 

We could add a foreign key to the Users database, but it would have to be optional